Content.Shared/Clothing/Components/ToggleableClothingComponent.cs

```diff
@@ -33,8 +33,9 @@ public sealed partial class ToggleableClothingComponent : Component
     /// <summary>
     ///     The inventory slot that the clothing is equipped to.
     /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
     [DataField("slot")]
-    public string Slot = "head";
+    public string Slot;
```

Comment:
Should this just use slotflags or the other enum.

Reply:
Right, enum is the perfect solution here, but I don't know all the slot names and wasn't able to find them. Any advice is welcome.

Reply:
Okay yeah SlotFlags is the one.

Reply:
I'm blind, it's literally SlotFlags, now it's done.

Reply:
Okay I ended up not liking this after opening up my IDE and looking further so I just reverted it on my end.